ejb3/src/main/resources/schema/wildfly-ejb4_0_0.xsd

```diff
@@ -0,0 +1,458 @@
+    </xs:complexType>
+
+    <xs:complexType name="statisticsType">
+        <xs:attribute name="enabled" type="xs:string"/>
+    </xs:complexType>
+
+    <xs:complexType name="channel-creation-optionsType">
+        <xs:annotation>
+            <xs:documentation>
+                <![CDATA[
+                The options that will be used while creating the channel for EJB remote invocation communication
+            ]]>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:choice minOccurs="0" maxOccurs="unbounded">
+            <xs:element name="option" type="optionType"/>
+        </xs:choice>
+    </xs:complexType>
+
+    <xs:complexType name="optionType">
+        <xs:annotation>
+            <xs:documentation>
+                <![CDATA[
+                The "name" attribute specifies the name of the option being configured.
+
+                The "value" attribute is the value that's going to be set for the option.
+
+                The "type" attribute value can either be "xnio" or "remoting". If it's "xnio", then the option
+                being configured will be looked up against the org.xnio.Options class. If it's "remoting" then
+                the option will be looked up against the org.xnio.Option.RemotingOptions class.
+            ]]>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="name" type="xs:string" use="required"/>
+        <xs:attribute name="value" type="xs:string"/>
+        <xs:attribute name="type" type="xs:string" use="required"/>
+    </xs:complexType>
+
+    <xs:complexType name="default-missing-method-permissions-deny-accessType">
+        <xs:annotation>
+                   <xs:documentation>
+                       <![CDATA[
+                       If this is enabled then lack of any security metadata for an EJB method is an
+                       implicit deny.
+                   ]]>
+                   </xs:documentation>
+               </xs:annotation>
+        <xs:attribute name="value" type="xs:boolean"/>
+    </xs:complexType>
+
+    <xs:complexType name="log-system-exceptionsType">
+        <xs:annotation>
+            <xs:documentation>
+                <![CDATA[
+                       If this is enabled then system exceptions will be logged by the EJB subsystem.
+                   ]]>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="value" type="xs:string"/>
+    </xs:complexType>
+
+
+    <xs:complexType name="disable-default-ejb-permissionsType">
+        <xs:annotation>
+            <xs:documentation>
+                <![CDATA[
+                       If this is enabled then the default security manager permissions required by spec will not
+                       be added to EJB deployments.
+                   ]]>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="value" type="xs:boolean"/>
+    </xs:complexType>
+</xs:schema>
```

Comment:
Brian commented on the previous PR which included other changes (WFLY-4853): "This is a change in the management API and the xsd since WF 9, so both need a major version bump."